konig-schemagen/src/test/java/io/konig/schemagen/sql/SqlTableGeneratorTest.java

```diff
@@ -144,7 +147,92 @@ public void testGCP() throws Exception {
 		
 		assertTrue(table!=null);
 		}
+	@Test
+	public void testSyntheticKeyForAwsAurora() throws Exception{
+		AwsShapeConfig.init();
+		MemoryShapeManager shapeManager = new MemoryShapeManager();
+		MemoryNamespaceManager nsManager = new MemoryNamespaceManager();
 		
+		ShapeLoader shapeLoader = new ShapeLoader(null, shapeManager, nsManager);
+		shapeLoader.loadTurtle(resource("aws/shape_PersonRdbmsShape.ttl"), null);
+		File baseDir = new File("target/test/resources/aws/sql");
+		baseDir.mkdirs();
+		DatasourceFileLocator sqlFileLocator = new DdlFileLocator(new File(baseDir.toString()));
+		CloudSqlTableWriter cloudSqlTableWriter = new CloudSqlTableWriter( new SqlTableGenerator(),sqlFileLocator);
+		cloudSqlTableWriter.visit(shapeManager.listShapes().get(0));
+		SqlTable table = generator.generateTable(shapeManager.listShapes().get(0));
+		
+		assertTrue(table!=null);
+		assertTrue(table.getColumnList()!=null && !table.getColumnList().isEmpty());
+		String query=table.toString();
+		for(SqlColumn column:table.getColumnList()){
+			if(SqlKeyType.SYNTHETIC_KEY.equals(column.getKeytype())){				
```

Comment:
If there is no column with a synthetic key, this test case should fail -- but it won't.

Instead of scanning for a column with the SYNTHETIC_KEY type, you should just get the expected column by name and then assert that it has the correct type.